source/PlayerInfo.cpp

```diff
@@ -1722,11 +1724,16 @@ bool PlayerInfo::TakeOff(UI *ui)
 	{
 		// Report how much excess cargo was sold, and what profit you earned.
 		ostringstream out;
-		out << "You sold " << Format::CargoString(sold, "excess cargo") << " for " << Format::CreditString(income);
-		if(totalBasis && totalBasis != income)
-			out << " (for a profit of " << Format::CreditString(income - totalBasis) << ").";
+		if(income) // we check against income for systems like Postverta where trading is disabled by having all prices at 0
```

Comment:
This comment should be on a separate line, with proper capitalization and a dot at the end.